quantum_chess/__init__.py

```diff
@@ -0,0 +1,3 @@
+# Quantum Chess no longer exists in this repo. See README.md for mre info.
+
+raise ImportError("quantum_chess moved to https://github.com/quantumlib/unitary/tree/main/unitary/quantum_chess")
```

Comment:
this should be under `recirq/quantum_chess` where the previous `__init__.py` file was